### PR TITLE
i18n titles

### PR DESCRIPTION
**Pull request by McGiverGim**
Some strings were translated, but the "titles" attribute were hardcoded in english. This PR translates this titles to be the in the same language as the value. I've changed all others titles of the code too.

A lot of others strings/values are hardcoded in english, with time I will continue the work to i18n all.

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -38,6 +38,24 @@
...
+    "sensorStatusGyro": {
+        "message": "Gyroscope"
+    },
+    "sensorStatusAccel": {
+        "message": "Accelerometer"
+    },
+    "sensorStatusMag": {
+        "message": "Magnetometer"
+    },
+"sensorStatusBaro": {
```
Comment (mikeller) [inline _locales/en/messages.json]: Indentation.

File: _locales/en/messages.json [hunk 2]
```diff
@@ -1503,6 +1524,59 @@
...
+    },
+    "motorsNumber7":{
+        "message": "Motor - 7"
+    },
+    "motorsNumber8":{
+        "message": "Motor - 8"
+    },
+
+    "servosNumber1":{
+        "message": "servo - 1"
```
Comment (wind0r) [inline _locales/en/messages.json]: i think this should be Servo not servo, as it was befor ;)
also i would prever servoNumber3 and motorNumber3 over servosNumber3 and motorsNumber3

Comment (McGiverGim) [reply _locales/en/messages.json]: No, in the "original code" it was 'servo - 1':

https://github.com/betaflight/betaflight-configurator/blob/5b5290c8ebb76591ea1ca8b450c182e884e0ed2b/tabs/motors.html#L130

I will change the motors and servos to the singular form.

Comment (McGiverGim) [reply _locales/en/messages.json]: True, I will change it.

**Comment by McGiverGim** [comment]
Changes done, and rebased against master. Thanks for the review!!

**Comment by mikeller** [comment]
This is ok for now. But we should probably look into generating things like the motor and servo list programmatically:

- there's no point in having 8 translations for 'motor' and a number;
- we actually support more than 8 motors, so if we do it dynamically, we can probably display more if required.

**Pull request merged**